### DISCUSSION TITLE
Not able to have any communication between background script and Quasar App

### DISCUSSION THREAD

**Discussion by daveavi** [quasarframework/quasar#8864] [CLI - BEX mode] [answered]
Hi! 

Currently, I am just getting myself introduced to developing a Quasar BEX. Right now I am following this tutorial: https://dev.to/quasar/quasar-and-browser-extension-development-getting-bexy-part-2-17ea , to help me understand how communication is done using Quasar. I am running into an issue where I am not able to render my extension onto the web page once the extension icon is clicked.
Here is my onClick listener in my background script: 
![image](https://user-images.githubusercontent.com/20130700/114431998-272f3a80-9b8e-11eb-88cf-28152201fc77.png)

Here is my main Vue Component that is listening for this call:
<img width="708" alt="Screen Shot 2021-04-12 at 12 28 06 PM" src="https://user-images.githubusercontent.com/20130700/114431379-7032bf00-9b8d-11eb-90f3-1fd062ef9c6f.png">

Here is my content script that should activate my Quasar application onto the web page:
<img width="591" alt="Screen Shot 2021-04-12 at 12 28 19 PM" src="https://user-images.githubusercontent.com/20130700/114431515-9193ab00-9b8d-11eb-8393-ff073c3e4873.png">

I have these log statements to see if communication is happening between all the parties involved, but the only thing being logged for me currently is in my background script logs where I am sending a message to my Quasar. Prior to this, I was sending a message directly to the content script, and it was working there, however that is not the best practice of using Quasar and I was running into issues with that too which is discussed in this thread : https://github.com/quasarframework/quasar/discussions/8843

Your help is much appreciated! 

Regards,

Avi Dave

**Comment by daveavi** [comment, marked as answer]
I was able to solve the issue, I made a silly error, by not using a IIFE in the content script to initially load my Vue component.

**Reply by realtebo** [reply]
if you could post a bit of code will help future googlers like me now
